otel/otel.sh

```diff
@@ -0,0 +1,79 @@
+# This initialization action installs OpenTelemetry Collector Contrib (otelcol-contrib) in a Dataproc cluster
+# configures OTel and pulls metrics from Prometheus endpoints provided by the customer. 
+# The metrics are then exported to Google Cloud Monitoring (GCM).
+
+set -euxo pipefail
+
+function install_otel() {
+  # Install otelcol-contrib as package (https://github.com/open-telemetry/opentelemetry-collector-contrib)
+  wget https://github.com/open-telemetry/opentelemetry-collector-releases/releases/download/v0.81.0/otelcol-contrib_0.81.0_linux_amd64.deb
+  dpkg -i otelcol-contrib_0.81.0_linux_amd64.deb
```

Comment:
You can pull the OTel version (`0.81.0`) as a readonly variable on top of this file.

Reply:
Added.